sdf/1.7/camera.sdf

```diff
@@ -5,7 +5,7 @@
     <description>An optional name for the camera.</description>
   </attribute>
 
-  <element name="camera_info_topic" type="string" default="camera_info" required="0">
+  <element name="camera_info_topic" type="string" default="__default__" required="0">
```

Comment:
I see that the `__default__` value for `<topic>` is handled internally here in sdformat:
https://github.com/gazebosim/sdformat/blob/sdf13/src/Sensor.cc#L240-L241

How about checking that here for `camera_info_topic`  instead of in gz-sensors?

Reply:
https://github.com/gazebosim/sdformat/pull/1201/commits/4a93c540061e65817488caa37680f4fdb6a1b3f8